uberfire-nio2-backport/uberfire-nio2-api/src/test/java/org/uberfire/java/nio/file/FilesTest.java

```diff
@@ -138,14 +138,13 @@ public void createDirectory() {
         final Path dir = Files.createDirectory(path.resolve("myNewDir"));
 
         assertThat(dir).isNotNull();
-        assertThat(dir.toFile().exists()).isTrue();
-        assertThat(dir.toFile().isDirectory()).isTrue();
+        assertThat(dir.toFile()).exists();
```

Comment:
@jhrcek by just invoing the methos... will `true` be checked?

Reply:
@porcelli before the change it was calling `exists` method of the `Path` class. Now it's calling `exists` of AbstractPathAssert [javadoc](http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractPathAssert.html#exists()) which gives better error messages in case the assertion fails. Previously it would give "expected true, but was false", now if it fails it would give something line "Expecting file [specific path] to exist".